src/Sarif.Viewer.VisualStudio/Sarif/CodeFlowLocation.extensions.cs

```diff
@@ -31,7 +31,7 @@ public static Microsoft.Sarif.Viewer.Models.CodeFlowLocationModel ToAnnotatedCod
                 model.LogicalLocation = codeFlowLocation.Location.FullyQualifiedLogicalName;
             }
 
-            model.Kind = codeFlowLocation.Kind.ToString();
+            model.Kind = "NODE_KIND";
```

Comment:
Why does `model` even have a `Kind`?

Reply:
This class is like a dummy view model so the VS designer will work. Missed removing that property...